.github/workflows/quality.yml

```diff
@@ -97,6 +102,25 @@ jobs:
               comment_id: ${{ steps.fc.outputs.comment-id }}
             })
 
+# Gradle dependency analysis using https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin
+  dependency-analysis:
+    name: Dependency analysis
+    runs-on: ubuntu-latest
+    # Allow all jobs on main and develop. Just one per PR.
+    concurrency:
+      group: ${{ github.ref == 'refs/heads/main' && format('ktlint-main-{0}', github.sha) || github.ref == 'refs/heads/develop' && format('ktlint-develop-{0}', github.sha) || format('ktlint-{0}', github.ref) }}
```

Comment:
I think you need to replace all occurence of `ktlint` by `dep` or something similar. I am not an expert on GHA though.

Reply:
Good catch, thank you! Fixed now.